test/support/helpers.rb

```diff
@@ -2,7 +2,13 @@
 
 class ActiveSupport::TestCase
   def cmd(name, env={}, as="someone")
-    post "/cogy/cmd/#{name}", env.merge("COG_CHAT_HANDLE" => as)
+    params = if Rails::VERSION::MAJOR >= 5
```

Comment:
Fix indentation in this conditional assignment.
```ruby
    params = if Rails::VERSION::MAJOR >= 5
               { params: env.merge("COG_CHAT_HANDLE" => as) }
             else
               env.merge("COG_CHAT_HANDLE" => as)
             end
```
http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/ConditionalAssignment